### PR TITLE
Fix PyPI action (hopefully)

### LINKED ISSUE #229: Bug in action for PyPI release
```
Run python setup.py sdist bdist_wheel
  python setup.py sdist bdist_wheel
  python -m twine upload --skip-existing dist/*
  shell: /usr/bin/bash -e {0}
  env:
    pythonLocation: /opt/hostedtoolcache/Python/3.10.12/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.10.12/x64/lib
    TWINE_USERNAME: __token__
    TWINE_PASSWORD: ***
/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/setuptools/config/_apply_pyprojecttoml.py:76: _MissingDynamic: `license` defined outside of `pyproject.toml` is ignored.
!!
        ********************************************************************************
        The following seems to be defined outside of `pyproject.toml`:
        `license = 'MIT'`
        According to the spec (see the link below), however, setuptools CANNOT
        consider this value unless `license` is listed as `dynamic`.
        https://packaging.python.org/en/latest/specifications/pyproject-toml/#declaring-project-metadata-the-project-table
        To prevent this problem, you can list `license` under `dynamic` or alternatively
        remove the `[project]` table from your file and rely entirely on other means of
        configuration.
        ********************************************************************************
!!
  _handle_missing_dynamic(dist, project_table)
/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/setuptools/config/_apply_pyprojecttoml.py:83: SetuptoolsWarning: `install_requires` overwritten in `pyproject.toml` (dependencies)
  corresp(dist, value, root_dir)
Traceback (most recent call last):
  File "/home/runner/work/sparkx/sparkx/setup.py", line 3, in <module>
    setup(
  File "/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/setuptools/__init__.py", line 104, in setup
    return distutils.core.setup(**attrs)
  File "/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/setuptools/_distutils/core.py", line 158, in setup
    dist.parse_config_files()
  File "/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/setuptools/dist.py", line 631, in parse_config_files
    pyprojecttoml.apply_configuration(self, filename, ignore_option_errors)
  File "/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/setuptools/config/pyprojecttoml.py", line 69, in apply_configuration
    return _apply(dist, config, filepath)
  File "/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/setuptools/config/_apply_pyprojecttoml.py", line 57, in apply
    _apply_project_table(dist, config, root_dir)
  File "/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/setuptools/config/_apply_pyprojecttoml.py", line 83, in _apply_project_table
    corresp(dist, value, root_dir)
  File "/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/setuptools/config/_apply_pyprojecttoml.py", line 184, in _license
    _set_config(dist, "license", val["text"])
KeyError: 'text'
Error: Process completed with exit code 1.
```

### PR DESCRIPTION
**Pull request by Hendrik1704**
This is try $n+1$ to fix the build and publish on PyPI action with $n\gg 1$.

This closes #229

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 setup(
     name='sparkx',
-    packages=find_packages(where='src/sparkx'),
-    package_dir={'': 'sparkx'},
+    packages=find_packages(where='src'),
```
[the remark this answers is not in the document]
Comment (Hendrik1704) [reply setup.py:5]: I don't know how to proceed with this. At least locally I'm able to build the package. Should we just try it or do you want to test another version for the next release?

[the remark this answers is not in the document]
Comment (NGoetz) [reply setup.py:5]: I was just wondering what of the advise is more correct. If you change the paths locally, like recommended here, does it then still work?

[the remark this answers is not in the document]
Comment (Hendrik1704) [reply setup.py:5]: This version gives an error

**Review by NGoetz — APPROVED** [review]
Let's hope this fixes it. Again.

**Pull request merged**